certified-connectors/D7Messaging/Readme.md

```diff
@@ -0,0 +1,57 @@
+# D7Messaging
```

Comment:
Please update readme.md file as mentioned in the template in this link. https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md

Reply:
Updated